ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -236,6 +236,10 @@ tests:
   steps:
     cluster_profile: azure4
     workflow: openshift-e2e-azure-sharednetwork
+- as: e2e-azure-existing-rg
```

Comment:
perhaps this should be renamed to `e2e-azure-resourcegroup` to match the naming convention of steps?